packages/markdown/parseMarkdown.js

```diff
@@ -159,7 +159,7 @@ function getHeaders(markdown) {
 function getContents(markdown) {
   return markdown
     .replace(headerRegExp, '') // Remove header information
-    .split(/^{{("(?:demo|component)":[^}]*)}}$/gm) // Split markdown into an array, separating demos
+    .split(/^\s*{{("(?:demo|component)":[^}]*)}}$/gm) // Split markdown into an array, separating demos
```

Comment:
Is there a way we could only allow indent for components? It would be great for this to not work with demos.

Reply:
Just to clarify, this allows indenting the input file, e.g. to allow

```md
* I'm a list
  {{...}}
  continue text here
```

if I can't indent the `{{` prettier turns this into:

```md
* I'm a list
{{...}}
continue text here
```

The indentation doesn't make it into the output, except that the component/demo can now be part of a list/callout/...

Reply:
@oliviertassinari We could do something like

```suggestion
    .split(/^\s*{{("component":[^}]*)}}|{{("demo":[^}]*)}}$/gm) // Split markdown into an array, separating demos
```

But quite honestly, I really don't see value in this complexity. I don't believe the semantic placement of demos should be the responsibility of the markdown parser.